operator/option/config.go

```diff
@@ -147,9 +147,9 @@ const (
 	// AzureResourceGroup is the resource group of the nodes used for the cluster
 	AzureResourceGroup = "azure-resource-group"
 
-	// UserAssignedIdentityName is the name of the user assigned identity used
+	// AzureUserAssignedIdentityID is the id of the user assigned identity used
 	// for retrieving Azure API credentials
-	UserAssignedIdentityName = "azure-user-assigned-identity-name"
+	AzureUserAssignedIdentityID = "azure-user-assigned-identity-id"
```

Comment:
it would be nice if we can add upgrade docs for this change.

Reply:
I did not add any upgrade docs because this is not released yet, still on `master` for both changes.